examples/13. Named Endpoints.md

```diff
@@ -6,6 +6,7 @@ This API example demonstrates how to define a standalone endpoint with an identi
 ## API Blueprint
 + [Previous: Advanced Action](12.%20Advanced%20Action.md)
 + [This: Raw API Blueprint](https://raw.github.com/apiaryio/api-blueprint/master/examples/13.%20Named%20Endpoints.md)
++ [Next: JSON Schema](14.%20JSON%20Schema.md)
```

Comment:
I don't know if JSON Schema has higher priority and should be listed a bit above in the examples directory. But that would mean editing a lot of examples. @zdne What is your directive on this? Should we just add the newer examples at the bottom?